C_CPP/Data Structures/Array/Equilibrium_index.md

```diff
@@ -0,0 +1,74 @@
+# Equilibrium Index of a Array
+
+Here we will see how to find a equilibrium index in a array.
+
+Equilibrium index means that index i such that the sum of all the lower indexes is same to the sum of the higher indexes.
+
+I have given a example to understand it better.
+
+## Example
+
+**Input**: Arr[] = {-7, 1, 5, 2, -4, 3, 0}
+
+**Output**: 3
```

Comment:
```suggestion
# Equilibrium Index of a Array
```suggestion
# Equilibrium Index of a Array

Here we will see how to find a equilibrium index in a array.

Equilibrium index means that index i such that the sum of all the lower indexes is same to the sum of the higher indexes.

I have given a example to understand it better.

## Example

**Input**: Arr[] = {-7, 1, 5, 2, -4, 3, 0}  
**Output**: 3
```
Here we will see how to find a equilibrium index in a array.  
Equilibrium index means that index i such that the sum of all the lower indexes is same to the sum of the higher indexes.  
I have given a example to understand it better.

## Example  
**Input**: Arr[] = {-7, 1, 5, 2, -4, 3, 0}  
**Output**: 3  
```